brewtils/plugin.py

```diff
@@ -604,6 +606,24 @@ def _handle_request_update_failure(self, request, headers, exc):
             )
             raise RepublishRequestException(request, headers)
 
+        elif isinstance(exc, TooLargeError):
+            self.logger.error(
+                "Error updating request {0} - the request exceeds the 16MB size "
+                "limitation. The status of this request will be marked as ERROR, but "
+                "it's possible the request actually completed successfully. If this "
+                "happens often please contact the plugin developer.".format(request.id)
+            )
+
```

Comment:
I have to create a new Request here because of the 2.x restriction on changing a Request status attribute. In 2.x this will raise its own exception:

```python
request = Request(status="SUCCESS")
request.status = "ERROR"
```

So I create a new one to get around this. In 3.x this is not an issue as that validation is only applied when persisting the request to the database.